config.sample.json

```diff
@@ -16,6 +16,7 @@
     "doorSensorPin": 16,
     "invertDoorState": false,
     "invertSensorState": false,
+    "input_pull": "off",
```

Comment:
Please change the default "input_pull" value to either "up" or "down" .  And leave a comment after "// options: "up" or "down".

Reply:
I think the correct configuration depends entirely on your hardware. In my case, I have a pull-up in hardware. One issue with any default value is that if you configure a pull that is opposite to what's in the hardware, then they cancel each other out and you wind up with a noisy input again.

Pulling the pin down and having a switch that also pulls it down results in it being impossible for the value to ever be high. Same with a pull-up and a switch that also pulls up.

At the end of the day, you just have to configure the thing as appropriate for your hardware.

I will add the comment, though. That's a good idea.

Reply:
Well, I've added it... but strictly speaking I have been told you're not actually allowed to have comments in JSON files. But since it's just a sample, and it's Javascript, it's probably ok.

Reply:
You are correct.  You cannot add a comment to a json file.  I'll merge the pull request and remove the comment.  I'll also update the README.md file with some more description.  Thank you for your help in this.  I'm sure others will appreciate it.  I've had your code in my RPi controlling my garage door all day.  Been in and out of the house multiple times and it works great.